src/spikeinterface/core/waveforms_extractor_backwards_compatibility.py

```diff
@@ -604,6 +609,21 @@ def _read_old_waveforms_extractor_binary(folder, sorting):
     return sorting_analyzer
 
 
+def make_ext_params_up_to_date(ext, old_params, new_params):
```

Comment:
this will not handle magically the nboefore to ms_before.
no ?

Reply:
This is an addition of @alejoe91 , i'll let him comment